controller_manager:
  ros__parameters:
    update_rate: 100 # Hz

    arm_controller:
      type: joint_trajectory_controller/JointTrajectoryController

    gripper_controller:
      type: joint_trajectory_controller/JointTrajectoryController

    joint_state_broadcaster:
      type: joint_state_broadcaster/JointStateBroadcaster



arm_controller:
  ros__parameters:
    joints:
      - base_joint
      - finger_joint
      - gripper_extension_joint
      - ee_joint
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity

gripper_controller:
  ros__parameters:
    joints:
      - gripper_tip_ee_right_joint
      - gripper_tip_ee_center_joint
      - gripper_tip_ee_left_joint     
    command_interfaces:
      - position
    state_interfaces:
      - position
      - velocity